tests/components/conftest.py

```diff
@@ -554,6 +554,7 @@ def _validate_translation_placeholders(
     full_key: str,
     translation: str,
     description_placeholders: dict[str, str] | None,
+    translation_errors: dict[str, str],
```

Comment:
This was the missing argument.
I've took the advantage of renaming it to avoid confusion.